### ISSUE TITLE
Append multiple rows to DataGrid at once

### ISSUE THREAD

**Issue by julie-puckett** [comet-ml/kangas#128]
I see that one at a time is possible, but is there a way to append multiple rows to a Data Grid at once?

**Comment by dsblank** [comment]
Yes, use `datagrid.extend(rows)` rather than `datagrid.append(row)` for each one.
